backend/routes/forms/submit.py

```diff
@@ -133,11 +133,65 @@ async def post(self, request: Request) -> JSONResponse:
                 response_obj.dict(by_alias=True)
             )
 
+            send_webhook = None
+            if FormFeatures.WEBHOOK_ENABLED.value in form.features:
+                send_webhook = BackgroundTask(
+                    self.send_submission_webhook,
+                    form=form,
+                    response=response_obj
+                )
+
             return JSONResponse({
-                "form": form.dict(),
+                "form": form.dict(admin=False),
                 "response": response_obj.dict()
-            })
+            }, background=send_webhook)
+
         else:
             return JSONResponse({
                 "error": "Open form not found"
             }, status_code=404)
+
+    @staticmethod
+    async def send_submission_webhook(form: Form, response: FormResponse) -> None:
+        """Helper to send a submission message to a discord webhook."""
+        # Stop if webhook is not available
+        if form.meta.webhook is None:
+            raise ValueError("Got empty webhook.")
```

Comment:
More of a question really, how will erroring out here affect things? Surely this would propagate up to a logged error.

Reply:
So the most probable cause for this to happen would be if the webhook feature was enabled, but the webhook link wasn't entered/was removed from the website. This might happen if we for example have an on/off toggle for the hooks, and a separate field to enter the URLs.

What this means, is that it is probably not an error, rather the form is just waiting on a URL to be setup. Ideally, this would be some sort of warning icon on the frontend. If we want to alert the user to an error happening (this would be more of a user error than a system error), we'd need to setup some sort of notification system on the frontend.

At the moment, it doesn't seem like much can be done for this, as there is no effective way to alert the user. Since the only people making forms right now already have access to the backend, and backend errors, this will hopefully non-fatally alert you while setting up forms.

Reply:
Gotcha, the feature and webhook URL will most likely be separate fields (e.g. enable webhook feature, then a text box is displayed).

We can probably validate on the frontend that there is not a point where the webhook feature is enabled when no webhook is set.